packages/paste-core/components/tooltip/src/index.tsx

```diff
@@ -40,36 +37,46 @@ export interface TooltipStateReturn extends TooltipPrimitiveStateReturn {
 }
 export interface TooltipProps extends TooltipPrimitiveInitialState {
   children: NonNullable<React.ReactElement>;
+  element?: string;
   state?: TooltipStateReturn;
   text: string;
 }
 
-const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(({baseId, children, state, text, ...props}, ref) => {
-  const tooltip = state || useTooltipPrimitiveState({baseId: `paste-tooltip-${useUID()}`, ...props});
-  return (
-    <>
-      {React.Children.only(
-        <TooltipPrimitiveReference {...tooltip} ref={ref} {...children.props}>
-          {(referenceProps) => React.cloneElement(children, referenceProps)}
-        </TooltipPrimitiveReference>
-      )}
-      <TooltipPrimitive {...tooltip} {...props} as={StyledTooltip}>
-        {/* import Paste Theme Based Styles due to portal positioning. */}
-        <StyledBase>
-          <TooltipArrow {...tooltip} />
-          <Text as="span" color="colorTextInverse">
-            {text}
-          </Text>
-        </StyledBase>
-      </TooltipPrimitive>
-    </>
-  );
-});
+/*
+Tooltip's current structure does not allow for customization of its arrow.
+TODO: refactor Tooltip so that the styling of its arrow can be customized
+using Customization Provider.
```

Comment:
This is fine, can you just make a ticket it and put it under the customization elements epic please?

Reply:
Done![ see here](https://issues.corp.twilio.com/browse/DSYS-3075)